Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbarSettings.h

```diff
@@ -5,6 +5,18 @@
 
 #include "SpatialGDKEditorToolbarSettings.generated.h"
 
+USTRUCT()
+struct FInteropTypebindingInfo
```

Comment:
Nice. Does this need to be a USTRUCT?

Reply:
They need to be there it needs to be visible in the editor